### PR TITLE
installation error after add b2

### PR DESCRIPTION
**Pull request by ctapmex**
Hi.
On clean install from master I had error
```
FAILED: pip3 install --upgrade b2sdk boto
-----------------------------------------
The directory '/home/***/.cache/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
The directory '/home/***/.cache/pip' or its parent directory is not owned by the current user and caching wheels has been disabled. check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
Collecting b2sdk
  Downloading https://files.pythonhosted.org/packages/50/4c/2e00d4f1e45692f66132c539edb3f9f3fc0a8938c6d49c151d1f72bb6fc2/b2sdk-1.2.0.tar.gz (172kB)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    ModuleNotFoundError: No module named 'setuptools'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-l1dici5k/b2sdk/
```

This PR fix it

### CODE REVIEW

**Comment by hija** [comment]
I cannot reproduce this bug. I just spinned up a brand new EC2 Ubuntu 18.04 instance on AWS and installed using the following commands:

```
git clone https://github.com/mail-in-a-box/mailinabox.git
cd mailinabox/
sudo ./setup/start.sh
```

Furthermore I see [that the recommended way to install setuptools is to install it using pip](https://setuptools.readthedocs.io/en/latest/userguide/quickstart.html).

Could you tell us which VPS provider you are using? I would like to see if I am able to reproduce the error myself :)

**Comment by ctapmex** [comment]
>Could you tell us which VPS provider you are using?
hetzner

**Comment by hija** [comment]
Okay, looks like Amazons Ubuntu 18.04 LTS differs from the version provided by hetzner. The hetzner's version seems to be more minimal, e.g. git is not installed by default. 

@ctapmex I would propose to install setuptools via pip3, since that's the "usual" way according to the documentation. Could you change your PR to do so?

@JoshData I think, this PR is important to be implemented before you release a new MIAB since it could break some boxes otherwise.

**Comment by ctapmex** [comment]
I checked base solution (without my patch) on DO - no error. 
I don`t insist on applying my patch because we can not test all OS images. You need to have a few base OS. In my fork, I'll fix it for myself.
I attached a more correct fix, checked it on hetzner - it works.

P.S. But there are still other errors during installation. hetzner image  not perfect for MIAB

**Review by hija — CHANGES_REQUESTED** [review]
I think, we should add wheel otherwise I still get one error on hetzner. But then the installation runs very smooth. I would check back on AWS if the installation still runs smooth and then I think it's ready to be merged 👍

File: setup/management.sh [hunk 1]
```diff
@@ -30,6 +30,7 @@ apt_install duplicity python-pip virtualenv certbot
 # b2sdk is used for backblaze backups.
 # boto is used for amazon aws backups.
 # Both are installed outside the pipenv, so they can be used by duplicity
+hide_output pip3 install --upgrade setuptools
```
[the remark this answers is not in the document]
Comment (ctapmex) [reply setup/management.sh]: ok, fixed

[the remark this answers is not in the document]
Comment (hija) [reply setup/management.sh]: One last thing: Could we use `--upgrade` as well? So we get always the newst version? :)

[the remark this answers is not in the document]
Comment (ctapmex) [reply setup/management.sh]: ok -)

**Review by hija — APPROVED** [review]

**Comment by JoshData** [comment]
My experience with pip-installing system-level packages like setuptools (and maybe wheel) is that you can run into problems if the package is also installed by the OS, e.g. via apt. Can these be installed by apt instead?

**Comment by hija** [comment]
Sure, I think the initial PR would work out then!

--> https://github.com/mail-in-a-box/mailinabox/pull/1899/commits/2aa0e28a9b6cdbc9f9b39bfe4ed0d9131f421126

**Comment by JoshData** [comment]
Done via the above commit.

**Pull request closed without merging**